sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/CertificatesTestBase.cs

```diff
@@ -217,6 +270,16 @@ protected void RegisterForCleanup(string certificateName)
             _certificatesToDelete.Enqueue(certificateName);
         }
 
+        protected void RegisterForCleanupIssuer(string issuerName)
+        {
+            _issuerToDelete.Enqueue(issuerName);
+        }
+
+        protected void RegisterForCleanUpContacts(List<CertificateContact> contacts)
```

Comment:
Nit: use most permissive type for parameters, e.g. `IEnumerable<CertificateContact>`, per .NET Guidelines.